drivers/entropy/entropy_nrf5.c

```diff
@@ -116,10 +116,12 @@ static int random_byte_get(void)
 	return retval;
 }
 
+#if defined(__GNUC__)
 #pragma GCC push_options
 #if defined(CONFIG_BT_CTLR_FAST_ENC)
```

Comment:
CONFIG_BT_CTLR_FAST_ENC  is stale Kconfig.... we should just remove this use of pragma and the Kconfig.

Reply:
oh I see. OK then, let's get rid of it.